packages/wonder-blocks-form/src/components/__tests__/labeled-text-field.test.tsx

```diff
@@ -382,28 +381,6 @@ describe("LabeledTextField", () => {
         expect(input).toBeInTheDocument();
     });
 
-    it("light prop is passed to textfield", async () => {
```

Comment:
We can remove the styling related unit tests since we'll have Chromatic tests to verify the styles!